src/sql/src/pure/error.rs

```diff
@@ -296,6 +296,11 @@ impl MySqlSourcePurificationError {
             Self::InvalidTableReference(_) => Some(
                 "Specify tables names as SCHEMA_NAME.TABLE_NAME in a FOR TABLES (..) clause".into(),
             ),
+            Self::UnrecognizedTypes { cols: _ } => Some(
+                "If the types are supported as text, use the TEXT COLUMNS option to \
```

Comment:
I think this will be somewhat unclear to users. What types are "supported as text" ?

Reply:
Good point -- in this case it'd be `json` and `enum` types, and presumably we'd have that information in our docs. Will adjust to make that clearer